src/profession.cpp

```diff
@@ -44,7 +72,11 @@ profession::profession()
 
 void profession::load_profession( JsonObject &jo, const std::string &src )
 {
-    all_profs.load( jo, src );
+    if( jo.has_array( "substitutions" ) ) {
+        item_substitutions.load( jo ); // This isn't actually a profession
```

Comment:
This looks like a dirty hack to me. Why not make it a proper type and load it separately like all the other types?